svc/env.go

```diff
@@ -87,15 +88,34 @@ func InitTracer() func() {
 //
 // Env Vars:
 // * STATSD_ADDR - The host:port of the statsd server.
-func InitMetrics() func() {
-	if addr := os.Getenv("STATSD_ADDR"); addr != "" {
-		metrics.SetEmpireDefaultTags()
-		metrics.Reporter, _ = metrics.NewDataDogMetricsReporter(addr)
+func InitMetrics() (fn func()) {
+	fn = func() {
+		metrics.Close()
 	}
 
-	return func() {
-		metrics.Close()
+	addr := os.Getenv("STATSD_ADDR")
```

Comment:
Maybe add a comment so future maintainers will realize why we're doing this? Said comment might be nice on a `resolveToIpAndPort` extracted method.